### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by fjdrjr**
updated and added German translation

### CODE REVIEW

File: chooloolib/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -34,4 +34,233 @@
...
     <string name="call_notification_channel_name">Anruf Benachrichtigung</string>
     <string name="call_notification_channel_description">Aktive Anrufbenachrichtigung, wird zur komfortablen Nutzung angezeigt</string>
-</resources>
\ No newline at end of file
+</resources>
+
+<resources>
+
+    <!--region titles-->
+    <string name="recents">Recents</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml]: Aktuelles oder Kürzlich

File: chooloolib/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -34,4 +34,233 @@
...
+    <string name="contacts">Kontakte</string>
+    <string name="actions">Actions</string>
+    <string name="conference">Conference</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml]: Konferenz

File: chooloolib/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="action_yes">Ja</string>
+    <string name="action_no">Nein</string>
     <string name="action_answer">Beantworten</string>
     <string name="action_hangup">Auflegen</string>
-    <string name="action_yes">Jap</string>
-    <string name="action_no">Nope</string>
-    <string name="hint_search">Suchen</string>
-    <string name="call_action_mute">Stummschalten</string>
-    <string name="call_action_unmute">Stummschaltung aus</string>
+    <string name="action_manage_calls">Manage Calls</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:32]:
```suggestion
    <string name="action_manage_calls">Anrufe verwalten</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="action_open_whatsapp">WhatsApp öffen</string>
+    <string name="action_block_number">Nummer blockieren</string>
+    <string name="action_delete_contact">Kontakt löschen</string>
+    <string name="action_delete_recent">Eintrag löschen</string>
+    <string name="action_set_favorite">Als Favorit markieren</string>
+    <string name="action_unset_favorite">Favorit entfernen</string>
+    <string name="action_open_in_whatsapp">In Whatsapp öffnen</string>
+    <string name="action_unblock_number">Blockierung der Nummer aufheben</string>
+    <string name="action_choose_audio_route">Audioausgang wählen</string>
+    <string name="action_choose_phone_account">Choose phone account</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:43]:
```suggestion
    <string name="action_choose_phone_account">Telefonkonto auswählen</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="hint_theme_mode">Design</string>
+    <string name="hint_sim_account">SIM-Karte</string>
+    <string name="hint_default_page">Startseite</string>
+    <string name="hint_animations">Animationen anzeigen</string>
+    <string name="hint_search_recents">Anrufliste durchsuchen</string>
+    <string name="hint_search_contacts">Kontakte durchsuchen</string>
+    <string name="hint_search_items">Kontakte &amp; Anrufliste durchsuchen</string>
+    <string name="hint_dialpad_vibrate">Wähtastenvibration</string>
+    <string name="hint_should_ask_sim">Vor jedem Anruf nach SIM fragen</string>
+    <string name="hint_dialpad_tones">Wähtastentöne</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:62]:
```suggestion
    <string name="hint_dialpad_tones">Wähltastentöne</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="hint_group_recents">Anrufeinträge einer Person gruppieren</string>
+    <string name="hint_show_blocked">Zeige Optionen zum Blockieren und Wiederfreischalten von Nummern (benötigt die Standardtelefonapp)</string>
+    <!--endregion-->
+
+    <!--region warning-->
+    <string name="explain_is_on_hold">%s wird gehalten</string>
+    <string name="explain_choose_theme_mode">Wähle das Koler-Design. App muss zum Anzeigen der Änderung neu gestartet werden.</string>
+    <string name="explain_choose_sim_account">Rufe mit einer anderen SIM an</string>
+    <string name="explain_choose_audio_route">Leite den Ton an einen anderen Ausgang um</string>
+    <string name="explain_choose_phone_account">Continue the current call using a specific phone account</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:72]:
```suggestion
    <string name="explain_choose_phone_account">Fortsetzen des laufenden Gesprächs unter Verwendung eines bestimmten Telefonkontos</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -1,37 +1,229 @@
...
     <string name="call_status_incoming">Eingehender Anruf</string>
-    <string name="call_status_disconnected">Anruf endete</string>
-    <string name="call_status_active">Anruf im gange</string>
-    <string name="pref_title_report_bugs">Fehler melden</string>
-    <string name="pref_title_email">Schreibe uns eine E-Mail</string>
-    <string name="pref_title_app_color">Akzent Farbe</string>
-    <string name="pref_title_animations">Animationen</string>
+    <string name="call_status_active">Anruf aktiv</string>
+    <string name="call_status_disconnected">Anruf beendet</string>
+    <string name="call_status_pulling_call">Pulling Call…</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:99]:
```suggestion
    <string name="call_status_pulling_call">Anrufe absetzen...</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 8]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="call_status_unknown">Anrufstatus unbekannt</string>
+    <string name="call_status_disconnecting">Trenne Verbindung…</string>
+    <string name="call_status_phone_account">Select phone account</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:102]:
```suggestion
    <string name="call_status_phone_account">Telefonkonto auswählen</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 9]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="content_description_favorite_star">Favoritenstern</string>
+    <string name="content_description_empty_list_icon">Empty list icon</string>
+    <!--endregion-->
+
+    <!--region type label-->
+    <string name="type_label_im">IM</string>
+    <string name="type_label_name">Name</string>
+    <string name="type_label_phone">Telefon</string>
+    <string name="type_label_email">E-Mail</string>
+    <string name="type_label_custom">Benutzerdfefiniert</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:221]:
```suggestion
    <string name="type_label_custom">Benutzerdefiniert</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 10]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="type_label_whatsapp">WhatsApp</string>
+    <string name="type_label_organization">Organisation</string>
+    <string name="action_call">Anrufen</string>
+    <string name="content_description_call_number">Nummer Anrufen</string>
+    <string name="content_description_add_contact">Kontakt hinzufügen</string>
+    <string name="content_description_open_contact">Kontakt hinzufügen</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:227]:
```suggestion
    <string name="content_description_open_contact">Kontakt öffnen</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 11]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="audio_route_speaker">Lautsprecher</string>
+    <string name="audio_route_earpiece">Ohrmuschel</string>
+    <string name="audio_route_bluetooth">Bluetooth</string>
+    <string name="audio_route_wired_headset">Gabelgebundenes Headset (mit Mikro)</string>
+    <string name="audio_route_wired_or_earpiece">Kabelgebundene Kopfhörer (ohne Mikro)</string>
+    <!--endregion-->
+
+    <!--region content description-->
+    <string name="content_description_favorite_star">Favoritenstern</string>
+    <string name="content_description_empty_list_icon">Empty list icon</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:213]:
```suggestion
    <string name="content_description_empty_list_icon">Symbol für eine leere Liste</string>
```

File: chooloolib/src/main/res/values-de/strings.xml [hunk 12]
```diff
@@ -1,37 +1,229 @@
...
+    <string name="error_no_permissions_phones">Berechtigung nötig, um Telefonnummern zu lesen :)</string>
+    <string name="error_no_permissions_recents">Berechtigung nötig, um die Anrufhistorie einzusehen :)</string>
+    <string name="error_no_permissions_contacts">Berechtigung nötig, um Kontakte einzusehen :)</string>
+    <string name="error_no_permissions_contacts_and_call_log">Berechtigungen nötig, um Kontakte und Anrufhistorie einzusehen :)</string>
+
+    <string name="error_no_number_to_call">Keine Nummer gefunden</string>
+
+    <string name="error_cant_hold_call">Anruf kann nicht gehalten werden</string>
+    <string name="error_cant_swap_calls">Anruf kann nicht gewechselt werden</string>
+    <string name="error_cant_merge_call">Can\'t merge call</string>
```
Comment (xdubx) [inline chooloolib/src/main/res/values-de/strings.xml:155]:
```suggestion
    <string name="error_cant_merge_call">Kann den Anruf nicht zusammenführen</string>
```

**Review by xdubx — COMMENTED** [review]
fix translation :)

**Comment by ngocanhtve** [comment]
I can contribute a translation for Vietnamese. Please show me the way! Thank you. Excuse me

**Pull request merged**